mindsdb/integrations/handlers/file_handler/file_handler.py

```diff
@@ -101,10 +138,37 @@ def query(self, query: ASTNode) -> Response:
             )
             result_df = query_df(df, query)
             return Response(RESPONSE_TYPE.TABLE, data_frame=result_df)
+
+        elif type(query) is Insert:
+            table_name = query.table.parts[-1]
+            file_path = self.file_controller.get_file_path(table_name)
+
+            # Load the existing data from the file
+            df, _ = self._handle_source(
+                file_path,
+                self.clean_rows,
+                self.custom_parser,
+                self.chunk_size,
+                self.chunk_overlap,
+            )
+
+            # Create a new dataframe with the values from the query
+            new_df = pd.DataFrame(query.values, columns=[col.name for col in query.columns])
+
+            # Concatenate the new dataframe with the existing one
+            df = pd.concat([df, new_df], ignore_index=True)
+
+            # Write the concatenated data to the file based on its format
+            format = Path(file_path).suffix.strip(".").lower()
+            write_method = getattr(df, f"to_{format}")
+            write_method(file_path, index=False)
```

Comment:
It seems to me it won't work in cloud because only local file is updated 
@StpMax, how is better to update file

Reply:
It should be fine. Of course will be need to test